src/main/java/com/launchdarkly/testhelpers/httptest/impl/JettyHttpServerDelegate.java

```diff
@@ -0,0 +1,108 @@
+package com.launchdarkly.testhelpers.httptest.impl;
+
+import com.launchdarkly.testhelpers.httptest.Handler;
+import com.launchdarkly.testhelpers.httptest.HttpServer;
+import com.launchdarkly.testhelpers.httptest.RequestContext;
+import com.launchdarkly.testhelpers.httptest.RequestInfo;
+import com.launchdarkly.testhelpers.httptest.ServerTLSConfiguration;
+
+import org.eclipse.jetty.server.Connector;
+import org.eclipse.jetty.server.HttpConfiguration;
+import org.eclipse.jetty.server.Request;
+import org.eclipse.jetty.server.SecureRequestCustomizer;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.ServerConnector;
+import org.eclipse.jetty.server.handler.AbstractHandler;
+import org.eclipse.jetty.util.ssl.SslContextFactory;
+
+import java.io.IOException;
+import java.security.KeyStore;
+import java.security.cert.Certificate;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Jetty implementation used by {@link HttpServer}.
+ * @author elibishop
```

Comment:
I'm guessing this tag was auto-added, and possibly undesired.

Reply:
Indeed I would be OK with not being forever known as the author of a small piece of Jetty code.